fast_api_als/routers/submit_lead.py

```diff
@@ -78,10 +93,21 @@ async def submit(file: Request, apikey: APIKey = Depends(get_api_key)):
 
     # check if vendor is available here
     dealer_available = True if obj['adf']['prospect'].get('vendor', None) else False
-    email, phone, last_name = get_contact_details(obj)
+
+    try:
+        email, phone, last_name = get_contact_details(obj)
+    except:
+        logging.error("/submit: fn submit : Error while getting contact details")
+        raise Exception("/submit: fn submit : Error while getting contact details")
+
```

Comment:
Since we've already validated the ADF, we've made sure that email, phone, and last_name exist.
so get_contact_details() won't be throwing any exceptions.

Reply:
It's good that you handled the exception here.